toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -113,41 +144,37 @@ func getFilesystemSizeInSectors(resize2fsStdout string, resize2fsStderr string,
 			resize2fsStderr)
 	}
 
-	blockCount, err := strconv.Atoi(match[1]) // Example: 21015
+	blockCount, err := strconv.ParseUint(match[1], 10, 64) // Example: 21015
```

Comment:
Is the switch from Atoi to allow handling of larger filesystem sizes?

Reply:
Yeah. There isn't a direct equivalent for `Atoi` that is 64-bit. You have to use `ParseUint` instead.